docs/file-reference.md

```diff
@@ -161,30 +161,55 @@ name: mariadb
 controller: deployment
 ```
 
+> Example using DeploymentConfig for OpenShift
+
+```yaml
+controller: deploymentconfig
+name: httpd
+replicas: 2
+containers:
+- image: centos/httpd
```

Comment:
this container won't work on OpenShift  (root privileges)

Reply:
you can use `bitnami/nginx` or `tomaskral/nonroot-nginx` (not sure if there is openshift's or redhat's official image)

Reply:
thanks. updated.